test/Staking.ts

```diff
@@ -199,5 +223,22 @@ describe("Staking contract", function () {
           accounts[3].address,
         ]);
     });
+
+    it("should be able to accept new validator after last one has unstaked", async () => {
```

Comment:
Typo, `should be able to accept a new......after the last one...`

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9